domains/qing762.json

```diff
@@ -4,8 +4,6 @@
         "email": "threatedblade@outlook.com"
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "qing-website.onrender.com"
```

Comment:
Unable to access site.

Reply:
It does. You need to wait for a minute or two. I dont have a paid plan on Render so that's what it does